README.md

```diff
@@ -107,23 +107,30 @@ make
 
 GTDynamics now supports a Pybind11-based Python API.
 
-To start, please download and install the [GTwrap repository](https://github.com/borglab/wrap).
+GTWrap comes with the installation with GTSAM, which can also be used for generate the python bindings for GTDynamics.
 
 To compile and install the GTDynamics python library:
 
-1. In the build directory, run `cmake` with the flag `GTDYNAMICS_BUILD_PYTHON=ON`.
+1. Ensure that GTSAM is built with generated python bindings. If not, go to the build directory and run `cmake` with the flag `GTSAM_BUILD_PYTHON=ON`. Afterwards, install the GTSAM python package.
+
+    ```sh
+    cmake -DGTSAM_BUILD_PYTHON=ON ..
+    make && make python-install
```

Comment:
You can just link to the install instructions in GTSAM directly. That should be more helpful in an overall context.

Reply:
I agree a link would be nice, but does not preclude including these 2 lines for convenience.

Reply:
Then the instructions should say `make && make install && make python-install`, since `make install` will actually install `gtwrap` for use.

Reply:
@varunagrawal https://github.com/borglab/gtsam/blob/develop/python/README.md#readme from this readme it looks like that `make python-install` will install everything including `gtwrap`, but not sure if it's outdated.